bedrock-lambda-cdk-dotnet/example-pattern.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "Extract text from images using Amazon Bedrock",
```

Comment:
```suggestion
    "title": "Automatically extract text from images in Amazon S3 using Amazon Bedrock",
```

Reply:
I'm only extracting newly uploaded image in S3 bucket. 
this one would be better?
"Automatically extract text from uploaded images in Amazon S3 using Amazon Bedrock."